mern/client/src/utils/startScreen.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+var res;
+
+async function uploadFlow(flow) {
```

Comment:
eventually need to move this file to frontend DAO once clean stucture is merged from my branch